content/departments/engineering/teams/core-services/managed-services/index.md

```diff
@@ -1,11 +1,18 @@
 # Managed Services
 
 A big part of the Core Services team is developing and maintaining our managed cloud services infrastructure.
+Going forward, new managed services - internal and external - should aim to be deployed using [MSP (Managed Services Platform)](./platform.md).
+
+## [Cody Gateway](../../cody/cody-gateway/index.md)
+
+> [!NOTE] Cody Gateway infrastructure is owned by the Core Services team, but the service is owned by #discuss-apis
 
 ## [Pings service](./pings.md)
 
+## [Telemetry Gateway](./telemetry-gateway.md)
```

Comment:
The way it's been put here feels like "Pings" is getting in the way of two gateways living together LOL

Reply:
hint at you for not naming it `pings gateway` 😉

Reply:
LOL true! too late to change? 😅